src/Riak/Command/Builder/ListObjects.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Basho\Riak\Command\Builder;
+
+use Basho\Riak;
+use Basho\Riak\Command;
+
+/**
+ * Used to list KV objects in Riak
```

Comment:
Can you add a disclaimer about the performance risks and not recommended for production use?